website/content/docs/release-notes/1.10.mdx

```diff
@@ -0,0 +1,109 @@
+---
+layout: docs
+page_title: 1.10
+description: |-
+  This page contains release notes for Vault 1.10.
+---
+
+# Vault 1.10 Release notes
+
+**Software Release Date**:
+
+**Summary**: Vault Release 1.10 offers features and enhancements that improve the user experience while closing the loop on key issues previously encountered by our customers. We are providing a summary of these improvements in these release notes.
+
+We encourage you to upgrade to the latest release to take advantage of the new benefits that we are providing. Additionally, with this latest release, we offer solutions to critical feature gaps that have been identified previously. For further information on product improvements, including a comprehensive list of bug fixes, please refer to the [Changelog](https://github.com/hashicorp/vault/blob/main/CHANGELOG.md) within the Vault 1.10 release.
+
+Some of these enhancements and changes in this release include:
+
+- The ability to provide a more granular approach to client billing.
+- Experiencing a secure posture that includes MFA for our Vault OSS customers
+- Achieving FIPS 140-3 and 140-2 compliance
```

Comment:
This note will need to be removed as we don't have FIPS 140-3 support yet and are still awaiting for confirmation regarding FIPS-140-2

Reply:
The section heading below about FIPS 140-2 inside should also be removed for the same reason.